assets/src/hooks/before_unload.ts

```diff
@@ -1,11 +1,15 @@
-const listener = (e: any) => {
-  e.preventDefault();
-  e.returnValue = '';
+const listener = (e: any, elementId?: string) => {
+  const element = elementId && document.getElementById(elementId);
+  if (!element || element.dataset.saved !== 'true') {
+    e.preventDefault();
+    e.returnValue = '';
+  }
 };
 
 export const BeforeUnloadListener = {
```

Comment:
The change shouldn't affect other places that it's being used because the only case in which the alert doesn't get shown is when the element to which the hook is attached has the `data-saved` attribute set to `"true"` and the manual_grading_view doesn't have that attribute. I checked if it was working correctly and it's showing the alert correctly